wdl/imputation.wdl

```diff
@@ -753,6 +762,14 @@ task plots {
         options(bitmapType='cairo')
 
         name <- "${name}"
+            if (file.exists("excluded_samples") & (file.size("excluded_samples") > 0)) {
```

Comment:
Is there a reason the same code is included twice? The file was already created in the previous R script?

Reply:
if I add it in only in one R code, the other R script gives errors.